### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by shavidissa**
Updated the README file as per the new format and removed the redundant files as they were moved to https://github.com/wavefrontHQ/wavefront-sdk-doc-sources/tree/master/python.

### CODE REVIEW

**Comment by shavidissa** [comment]
CC @rkempf42

File: README.md [hunk 1]
```diff
@@ -191,6 +314,8 @@ batch_metric_data = [one_metric_data, one_metric_data]
 # Send list of data immediately
 wavefront_sender.send_metric_now(batch_metric_data)
 ```
+***Note***: If your metric name has a bad character, that character is replaced with a `-`.
```
Comment (rkempf42) [inline README.md:317]: Maybe: ...the proxy replaces that character with a `-`. 
?

File: README.md [hunk 2]
```diff
@@ -246,31 +373,46 @@ batch_span_data = [one_tracing_span_data, one_tracing_span_data]
 wavefront_sender.send_span_now(batch_span_data)
 ```
 
-## Get a Failure Count
+## Get them Failure Count
```
Comment (rkempf42) [inline README.md:376]: Get the Failure Count

File: README.md (hunk 1 above)
Comment (shavidissa) [reply README.md:317]: Checked with Aliaksandrand he mentioned that this is done by the SDK itself, not the proxy.

File: README.md (hunk 2 above)
Comment (shavidissa) [reply README.md:376]: Thank youuu!

**Comment by shavidissa** [comment]
CC @vikramraman

**Pull request merged**